keyboards/dtd/ldmlKeyboard3.dtd

```diff
@@ -0,0 +1,158 @@
+<!--
+Copyright © 1991-2022 Unicode, Inc.
+For terms of use, see http://www.unicode.org/copyright.html
+SPDX-License-Identifier: Unicode-DFS-2016
+CLDR data files are interpreted according to the LDML specification (http://unicode.org/reports/tr35/)
+-->
+<!-- Important Note:
+
+The CLDR Keyboard Workgroup is currently developing major changes to the
+CLDR keyboard specification.
+
+This DTD is a work in progress.
+
+Please see CLDR-15034 for the latest information. -->
+
+<!ELEMENT keyboard ( version, info?, names, settings?, import*,
+	vkeyMaps*, displayMap*, keyMap+, layerMaps,
+	transforms*, reorders?, backspaces?,
+	special* ) >
+<!ATTLIST keyboard locale CDATA #REQUIRED >
+    <!--@MATCH:any/TODO-->
+<!ATTLIST keyboard conformsTo CDATA #REQUIRED >
+	<!--@METADATA-->
+    <!--@MATCH:version-->
+
+<!ELEMENT version EMPTY >
+<!ATTLIST version number CDATA #IMPLIED >
+    <!--@METADATA-->
+<!ATTLIST version cldrVersion CDATA #FIXED "42" >
+    <!--@MATCH:version-->
+    <!--@METADATA-->
+
+<!ELEMENT info EMPTY >
+<!ATTLIST info author CDATA #IMPLIED >
+    <!--@VALUE-->
+<!ATTLIST info normalization CDATA #IMPLIED >
+    <!--@MATCH:literal/NFC, NFD, other-->
+    <!--@VALUE-->
+<!ATTLIST info layout CDATA #IMPLIED >
+    <!--@VALUE-->
+<!ATTLIST info indicator CDATA #IMPLIED >
+    <!--@VALUE-->
+
+<!ELEMENT names ( name+, special* ) >
+
+<!ELEMENT name EMPTY >
+<!ATTLIST name value CDATA #REQUIRED >
+    <!--@MATCH:any-->
+    <!--@VALUE-->
+
+<!ELEMENT settings EMPTY >
+    <!--@ORDERED-->
+<!ATTLIST settings fallback (omit) #IMPLIED >
+    <!--@VALUE-->
+<!ATTLIST settings transformFailure (omit) #IMPLIED >
+    <!--@VALUE-->
+<!ATTLIST settings transformPartial (hide) #IMPLIED >
+    <!--@VALUE-->
+
+<!ELEMENT import EMPTY >
+<!ATTLIST import path CDATA #REQUIRED >
+
+<!ELEMENT keyMap ( map+, special* ) >
+<!ATTLIST keyMap modifiers CDATA #IMPLIED >
```

Comment:
Is `modifiers` appropriate now for `keyMap`? I thought that was controlled by `layerMap`?

Reply:
```suggestion
```

No, should be removed.